lms/templates/commerce/provider.underscore

```diff
@@ -0,0 +1,13 @@
+<p class="bold_param">
+  <%= interpolate(gettext("You still need to see visit %s to complete the credit process."), [interpolate("<a href='%s'>%s</a>", [provider_url, display_name]) ]) %>
+  <span><%= interpolate("<img src='%s' alt='%s'></image>", ["", display_name]) %></span>
+</p>
+<p>
+  <%= interpolate(gettext("In order to learn credit, %s requires learner to:"), [display_name]) %>
+</p>
+<div class="custom_instructions">
+  <div>
+    <%- fulfillment_instructions %>
+  </div>
+  <%= interpolate('<a href="%s" class="complete-course">%s</a>', [provider_url, gettext( "Complete Order")]) %>
```

Comment:
I don't think this is quite right.  After the user has paid for credit, she still needs to request credit from the credit provider.  This means that:

a) We need to create a record of the user's request, and
b) We need to construct the parameters to POST to the credit provider so they know who the user is.

There are already API calls that do this.  You should be able to re-use the JavaScript that @ahsan-ul-haq wrote when he implemented this on the student dashboard: https://github.com/edx/edx-platform/pull/8776

In particular, you can adapt this code, which handles the button click and error display: https://github.com/edx/edx-platform/blob/c94f043c2b6ea1585d1415350a7f7513bb0abb44/lms/static/js/dashboard/credit.js#L23-L65